### PR TITLE
chore(prometheus/changelog): exposing tags as metric label

### PR DESCRIPTION
**Pull request by carnei-ro**
### Description

Add entries in Prometheus' Changelog ref to https://github.com/Kong/kong/pull/10082

### Testing instructions

Netlify link: 

### Checklist 

- [ ] Review label added 
- [ ] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_hub/kong-inc/prometheus/_1.6.0.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/prometheus/_1.6.0.md:344]:
```suggestion
* Added the `expose_tags.from_service`, `expose_tags.from_route`, and `expose_tags.from_consumer` configs so you can expose services, routes, or consumers tags as metrics with the `tags` label.
```
@carnei-ro Does this revision sound accurate?

File: app/_hub/kong-inc/prometheus/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/prometheus/_index.md:295]:
```suggestion
* Added the `expose_tags.from_service`, `expose_tags.from_route`, and `expose_tags.from_consumer` configs so you can expose services, routes, or consumers tags as metrics with the `tags` label.
```

**Review by cloudjumpercat — COMMENTED** [review]
Thanks for updating this doc! I just added a couple revisions if you want to take a look at those. I marked this one as "blocked:kong" because I'll merge this after the PR you opened in kong/kong is merged.

**Comment by carnei-ro** [comment]
@cloudjumpercat Thanks for reviewing!

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Comment by cloudjumpercat** [comment]
I'll close this PR since the related kong/kong PR (https://github.com/Kong/kong/pull/10082) was also closed due to inactivity and performance issues. This can be reopened if there's any activity on the old PR.

**Pull request closed without merging**
